docs/swagger.yaml

```diff
@@ -341,6 +469,61 @@ components:
           example: 'https://example.com/myProduct'
       required:
         - redirectUrl
+    MerchantCallbackPutRequest:
+      title: MerchantCallbackPutRequest
+      x-stoplight:
+        id: 56a28c70438c7
+      type: object
+      x-examples:
+        Example 1:
+          location: Kasse 1
+      properties:
+        location:
+          type: string
+          description: 'A description of where the QR code will be located. It will be shown in the app when a user scans the QR code. Examples could be ‘Kasse 1’ , ‘Kiosk’ or ‘Platform 3’.'
+      required:
+        - location
+    MerchantCallbackMobilePayPutRequest:
+      title: MerchantCallbackMobilePayPutRequest
+      type: object
+      x-examples:
+        Example 1:
+          location: Kasse 1
+      properties:
+        location:
+          type: string
+          description: 'A description of where the QR code will be located. It will be shown in the app when a user scans the QR code. Examples could be ‘Kasse 1’ , ‘Kiosk’ or ‘Platform 3’.'
+      required:
+        - location
+      x-stoplight:
+        id: 574e5e977f056
+    MerchantCallbackQr:
```

Comment:
maaaaybe we should consider adding the qrUrl to this? If the merchants have a box of 50 QRs and need to figure out which does what, they'll need a way of mapping our hex'e (from the url in the QR) to their internal id's.

either that or we could have a /resolve endpoint where they could check?

Reply:
I would hope that we did not need to respond with the hex. I hope the location property would help them to know what QR code needs to map to where in the store. We could maybe somehow print the location onto the printed QR code. No that its part of the code itself, but just be printed next to the sticker so they can see the location property when they need to stick the QR code in the store.
Also, do we wanna send the HEX in the callback to the merchant?

Reply:
We dont want to send the hex to merchant, no. 
We have discussed a design where we veeery barely print the merchantQrId on the bottom of the sticker.

How do you solve this in mobilepay today? Are there any ids on the printed stickers?

Reply:
Thats actually a good question. I am pretty sure we print the BeaconId which would kind of be the equivalent of the MerchantQrId here. The benefit in MobilePay is that the beaconId is also part of the url. So if they scan the QR code they can see the beaconId from the URL. Also, the beaconId is mapped to a PoS concept that the integrator keeps track of, so they already have a mapping between the beaconId and a name of the ECR. For example 'Kasse 1'. So that helps them as well. But if the merchants are responsible for the mapping of merchantQrIds and we require them to input a location property that tells where the QR is to be placed in the store, then they should be able to see where a QR should be sticked if they have the merchantQrId printed along with the sticker

Reply:
mhhhm, makes sense. In our case we either have to print the id, or supply the merchant with a way to figure out what the hex means

Reply:
We decided to return the qrContent (hex) to merchants so they can map what they scan to which merchantQrId that belongs to